config/locales/en.yml

```diff
@@ -223,6 +223,14 @@ en:
       additional_code_description_label_html: "<strong>Additional code description</strong>"
       additional_code_description_hint_text: "Use this field if you know a key phrase included in the additional code description."
       submit: "Search"
+  footnote_search:
+    form:
+      legend_text: "Search"
+      footnote_label_html: "<strong>Footnote code</strong>"
```

Comment:
I guess we should avoid storing html tags in the locales

Reply:
I'm never sure about this. I think a single strong tag isn't painful and locales have a built in idiom for converting html locales to be safe with the `_html` key suffix.

Reply:
Ok, I think we should not store them, ... but we are already doing it, so it's fine 😄